docs/patterns/clustered-listening-tentacles.md

```diff
@@ -0,0 +1,151 @@
+position: 0
+---
+
+You can configure a pair of Octopus Tentacles in an Active/Passive failover cluster using shared storage with the **Failover Cluster Manager**. This approach has been tested on Windows Server 2016. 
+
+To perform this task, first ensure that a shared drive is available to the node where you are installing the new Octopus Tentacle instance. Once all nodes have the Octopus Tentacle instances installed, pick any node to generate and export a private key. Proceed to import the new private key onto every Octopus Tentacle instance, including the node which was used to export it. 
+
+Using the Failover Cluster Manager you then configure the Octopus Tentacle instance service as a generic service, ensuring it has a clustered IP and gets registered in DNS. Once the Tentacles have been setup, use the Octopus Server to discover the new Tentacle cluster on the Clustered IP address specified when making Octopus Tentacles a generic service cluster. 
+
+:::warning
+**Shared Storage Considerations**
+It is not possible to store the **tentacle.config** file in shared storage because the Tentacle.Certificate component gets encrypted using a node's machine-specific key. If you attempt to store the tentacle.config file in shared storage, you will encounter this error: **Key not vaild for use in specified state** upon switching to a new active node. This occurs because the new active node is not able to decrypt the private key which will result in the Tentacle service failing to start.
+:::
+
+To avoid any issues with a node's private key you must specify an alternate location for the **tentacle.config** file upon installation of the Tentacle service. Follow these instructions to install Octopus Tentacle in a failover cluster using shared storage.
+
+## Requirements {#ClusteringTentacles-Requirements}
+
+This document assumes you already have the following setup:
+
+- An Active Directory Domain and a local DNS server.
+- An Octopus Server, this does not need to be joined to the domain.
+- A two node active/passive windows cluster where each node is joined to the domain.
+- A local IP Address available for the cluster so that the Octopus Server can reach the cluster.
+- Shared Storage configured for the cluster, this example has mounted it as E:\
+
+## Installation {#ClusteringTentacles-Installation}
+
+:::warning
+**Installing Tentacles with Shared storage**
+This document implements shared storage using an iSCSI target with Multipath IO confgured on the Tentacle servers, in this scenario it is best to avoid accessing the same iSCSI volume from two different hosts at the same time.
+:::
+
+On the first node in your Active/Passive Windows cluster, check that the shared drive is mounted by seeing that it is available in disk management and note down the Drive letter. If it hasn't already been done, bring the shared disk online and format it with a drive letter, we have chosen to use E:\
```

Comment:
`in your Active/Passive Windows cluster` is unnecessary
`by seeing that it is available` is unnecessary
`If it hasn't already been done, bring the shared disk online and format it with a drive letter, we have chosen to use E:\` => not needed.